reports/2023-project-director-elections-retrospective.md

```diff
@@ -0,0 +1,112 @@
+# 2023 Project Director Elections Retrospective
+
+One of the goals for the Project Director Election Process Committee is to "[Track, review, and document the effectiveness of the process][goals]."
+Now that we've completed the election process, we'd like to reflect on how things went and make recommendations for future elections.
+
+[goals]: ../committees/project-director-election-process.md
+
+## Summary
+
+We were able to elect three new Project Directors and we did so in time to have them attend the first Board meeting after the outgoing directors' terms ended.
+These were chosen from a large pool of candidates that were nominated from across the project.
+
+Going forward, we'd recommend working to improve the following areas:
+
+1. **Candidate Eligibility and Requirements** - We had late-breaking questions around how "employment" is interpreted for the purposes of employment limits and around the legal requirements for candidates to disclose their names. At a minimum, we need to clearly document these requirements. We should also consider whether changes to these requirements are possible and desirable, as they significantly constrain who is able to serve in this role and makes it difficult to achieve other goals around diversity.

```

Comment:
This seems like a concrete task. Can we make sure we capture this as an issue?

Reply:
I opened #66.